### DISCUSSION TITLE
[Feature request] Add Android auto contols for playback speed.

### DISCUSSION THREAD

**Discussion by CptanPanic** [PaulWoitaschek/Voice#1878] [Ideas]
Doesn't seem like there is a way to adjust playback speed using Android auto.

**Comment by BrianPeter99** [comment]
+1 on this request, it's the only thing keeping this from being a 5 star app.

**Reply by BOB74j** [reply]
Yes! I'm not sure the current sleep timer option is of any benefit in Android Auto. Playback speed control would be nice to see. I can only focus on 1.2x speed or so when there is traffic, but driving on open highway I can listen at higher speeds.

I have seen other audiobook and podcast apps use a playback speed button in the Android Auto interface to cycle between a few different speed presets such as 1x, 1.25x, 1.5x, 1.75x, 2.0x, and 0.75x. Perhaps a similar solution could be implemented in Voice.

**Comment by LeonidRosin** [comment]
I also miss changing the playback speed in Android Auto mode.
And I also think the sleep timer is useless in Android Auto.
